chainio/clients/wallet/fireblocks_wallet.go

```diff
@@ -118,18 +130,40 @@ func (t *fireblocksWallet) SendTransaction(ctx context.Context, tx *types.Transa
 	if err != nil {
 		return "", fmt.Errorf("error getting whitelisted contract %s: %w", tx.To().Hex(), err)
 	}
+
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	// if the nonce is already in the map, it means that the transaction was already submitted
+	// we need to get the replacement transaction hash and use it as the replaceTxByHash parameter
+	replaceTxByHash := ""
+	nonce := tx.Nonce()
```

Comment:
so the idea is that I would send the same tx but with a gas price increase? and the nonce here would be the same as the previous tx because it still hasn't been mined so when abigen creates the tx it gets the old nonce?

Reply:
Yes, this method doesn't change any parameters, so the increased gas price should be specified in the tx. This is the same interface you would normally use chain client to replace a transaction.